pkg/selinux/selinux-client.js

```diff
@@ -108,8 +110,18 @@ export function init(statusChangedCallback) {
         // Example:
         // authlogin_nsswitch_use_ldap (on , on) Allow authlogin to nsswitch use ldap
         // Split by first ')', as the name cannot contain ')'
-        if (item)
-            result.push(item.split(")", 2)[1].trim());
+        if (item) {
+            const description = item.split(")", 2)[1].trim();
+            const state = item.split("(", 2)[1].startsWith("on") ? "yes" : "no";
```

Comment:
This is rather brittle with whitespace changes. It would be more robust and also easier to read (IMHO) with using groups in a regular expression, and allowing arbitrary whitespace pretty much everywhere.